google/gax/config.py

```diff
@@ -71,3 +71,7 @@ def exc_to_code(exc):
 
 create_stub = grpc.create_stub  # pylint: disable=invalid-name,
 """The function to use to create stubs."""
+
+
+ABORTION_ERRORS = (face.AbortionError, )
```

Comment:
Done, and I will follow this change with one to move the rest of config.py into grpc.py